lib/matplotlib/figure.pyi

```diff
@@ -25,6 +25,9 @@ from matplotlib.lines import Line2D
 from matplotlib.patches import Rectangle, Patch
 from matplotlib.text import Text
 from matplotlib.transforms import Affine2D, Bbox, BboxBase, Transform
+if TYPE_CHECKING:
```

Comment:
Actually, this guard is not needed in .pyi files, because they are only used for tyoechecking.

Reply:
OK, removed